client/components/settings/settingBody.js

```diff
@@ -0,0 +1,132 @@
+Meteor.subscribe('setting');
```

Comment:
The reason why I use "setting" instead of "settings" is that I want to make it clear that it publishes a singleton to the client, not a set of something, it's more like "settings.public", "settings.singleton", or something like that.